common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1615,6 +1615,9 @@
   "function.wynntils.accessoryDurability.name": "Accessory Durability",
   "function.wynntils.allShinyStats.description": "Shows a list of all shiny stats",
   "function.wynntils.allShinyStats.name": "All Shiny Stats",
+  "function.wynntils.amountOfItems.argument.name": "The name of item to count for instead",
+  "function.wynntils.amountOfItems.description": "Get the amount of items in total in your inventory",
+  "function.wynntils.amountOfItems.name": "Amount of Items",
```

Comment:
"suggestion": {
  "function.wynntils.amountOfItems.argument.name": "The name of the item to count",
  "function.wynntils.amountOfItems.description": "Returns the total number of items with the specified name in your inventory",
  "function.wynntils.amountOfItems.name": "Item Count"
}

Reply:
Changed it with your suggestion and to be a bit more accurate, is this better?

Reply:
I think my description is a bit more whole.

Reply:
I mean the thing is, the function with no arguments will match all items, the argument is optional, when provided it will check if the name of the item ends with that argument, so for example if you put "]" as the argument, it would count all ingredients, teleportation scrolls and everything that ends with "]". Your description makes it sound like the name argument is mandatory, and you need to provide the whole name of the item.